docs/CMakeLists.txt

```diff
@@ -44,6 +44,21 @@ if(NOT DOC_VERSION)
 	endif()
 	string(STRIP "${stagever}" stagever)
     set(DOC_VERSION "${version}")
+else()
+	set(version "${DOC_VERSION}")
+	string(STRIP "${version}" version)
+	execute_process(
+	COMMAND "grep" "-m 1" "<version>" "${CMAKE_CURRENT_SOURCE_DIR}/../pom.xml"
+	COMMAND "sed" "s/[ ]*<version>[0-9]*.[0-9]*.[0-9]*-//"
+	COMMAND "sed" "s/[-][^0-9]*<\\/version>$//"
+	WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+	OUTPUT_VARIABLE stagever
+	RESULT_VARIABLE stagever_rv
+	)
+	if("${stagever_rv}" GREATER "0")
```

Comment:
this confuses me, in effect stagever_rv can only be 0 or negative? is that somehow accounting for the '-' delimiter?